compiler/ballerina-lang/src/main/java/io/ballerina/projects/CompilationOptions.java

```diff
@@ -93,4 +100,11 @@ private boolean toBooleanDefaultIfNull(Boolean bool) {
         }
         return bool;
     }
+
+    private String toStringDefaultIfNull(String value) {
+        if (value == null) {
+            return "";
```

Comment:
We can set a default value with the flag in the BuildCommand. Check suggestion https://github.com/ballerina-platform/ballerina-lang/pull/27949/files#r560115018

Reply:
We can avoid the null handling by doing so

Reply:
We cant do this since the option can also be passed through the Ballerina.toml